src/App.js

```diff
@@ -1,8 +1,7 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
 
-const Logo = () => <img src={logo} className="App-logo" alt="logo" />;
+import { Logo } from './shared/components/Logo';
```

Comment:
@ewal2 moved the logo component to a separate file

Reply:
I see